docs/source/_static/custom.css

```diff
@@ -1,3 +1,8 @@
 td {
     padding: 0px 10px 0px 10px;
 }
+
+img.align-center {
```

Comment:
Comment: I wonder if in the future we will need to create a special class for image examples. This will require us to have our own custom sphinx I think... In any case, let's go with this for now.

Reply:
Yes creating classes is definitely needed. But yeah for now this good.

Reply:
In the long run, it might be necessary, especially if we want the images / animations to behave in a special way. Let's keep it in mind while extending the documentation.

Reply:
Also I plan to add some js things for site optimization. Yes if I find any time.